github/gists.go

```diff
@@ -64,6 +64,7 @@ func (gc GistCommit) String() string {
 	return Stringify(gc)
 }
 
+// GistFork represents a fork of a gist
```

Comment:
Nitpick. This sentence does not end with a period. Other comments have sentences that do end with periods, so it'd be nice and more consistent to include a period at the end of this sentence too.